web-src/src/index.css

```diff
@@ -133,4 +133,56 @@ body {
 .variant-image-button {
   opacity: 0;
   transition: opacity 0.3s ease-in-out 0.1s;
-}
\ No newline at end of file
+}
+
+.slick-prev:before {
+  transform: rotate(180deg);
+}
+
+.slick-prev:before, .slick-next:before {
+  opacity: 1;
+  content: '';
+  width: 64px;
+  height: 64px;
+  display: block;
+  position: absolute;
+  top: 0;
+  background-size: contain;
+  background: url('./assets/variations-scroll-indicator.svg');
+}
+.slick-arrow {
+  background: #fff;
+  z-index: 2;
+  border-radius: 50%;
+  width: auto;
+  height: auto;
+  filter: opacity(0);
+}
+
+.slick-slider:hover .slick-arrow {
+  filter: opacity(1);
+}
```

Comment:
Can we slightly ease the hover effect by transitioning the buttons from opacity 0 to 1?
`transition: opacity 0.3s ease-in-out 0.1s;`

Reply:
@askayastha22  It should be fine now :-)
![Screen Shot 2024-05-10 at 5 04 02 PM](https://github.com/adobe/aem-genai-assistant/assets/7125318/5502b2b4-da75-4c1c-b19f-7a9eea6e27a2)